ddl/ddl_api.go

```diff
@@ -1579,7 +1579,12 @@ func (d *ddl) CreateTableWithInfo(
 		if tbInfo.AutoIncID > 1 {
 			// Default tableAutoIncID base is 0.
 			// If the first ID is expected to greater than 1, we need to do rebase.
-			err = d.handleAutoIncID(tbInfo, schema.ID)
+			err = d.handleAutoIncID(tbInfo, schema.ID, autoid.RowIDAllocType)
+		}
+		if tbInfo.AutoRandID > 1 {
```

Comment:
Should  check the error in line 1582 first? otherwise, the error maybe override by line 1587?

Reply:
oh! yes, addressed